Cargo.toml

```diff
@@ -48,10 +48,14 @@ time = "0.3.16"
 unicode-width = "0.1.9"
 url = "2.2.2"
 
+[patch.crates-io.reqwest]
+git = "https://github.com/seanmonstar/reqwest"
+branch = "master"
```

Comment:
Perhaps, I think this should be removed before publishing on [crates.io](https://crates.io/).

> Note: [crates.io](https://crates.io/) does not allow packages to be published with dependencies on code published outside of [crates.io](https://crates.io/) itself ([dev-dependencies](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#development-dependencies) are ignored). See the [Multiple locations](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#multiple-locations) section for a fallback alternative for git and path dependencies.[^1]

[^1]: <https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-path-dependencies>

Reply:
Once the new version of reqwest is released, it will be removed.

This PR will not merge into master before the new version of reqwest is released.